.github/workflows/release.yml

```diff
@@ -0,0 +1,18 @@
+name: Release
+on:
+  push:
+    tags: [ v* ]
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby-version }}
```

Comment:
Will this work ? matrix is not defined in the release.yml file

Reply:
Fixed